defaults/main.yml

```diff
@@ -3,6 +3,10 @@
 # Version numbers from Nodesource: https://github.com/nodesource/distributions
 nodejs_version: "0.12"
 
+# nodejs_install_npm_user: username
+
```

Comment:
Nevermind I looked at the ruby role where a fact is set. That's missing here. https://github.com/geerlingguy/ansible-role-ruby/blob/e2262955f6a458919462b9d8025c0ead004b3b7e/tasks/main.yml#L17:L20